lib/services/amazonka-s3/src/Amazonka/S3/Internal.hs

```diff
@@ -211,9 +220,9 @@ _ObjectKey = iso (\(ObjectKey k) -> k) ObjectKey
 --
 -- >>> "/" ^? keyPrefix '/'
```

Comment:
Do these haddocks need to be updated to match the name changes?

Reply:
Yes, they do.